### ISSUE TITLE
Port 10050

### ISSUE THREAD

**Issue by LordBeaver** [HeinleinSupport/olefy#1]
Port 10050 is already in use by Zabbiz Agent. Probably need to change the default configuration file, until the product has become popular.

**Comment by c-rosenberg** [comment]
Definitely, 10050 was a bit headless. We will change this maybe with the next Rspamd major release
